facility-manager.html

```diff
@@ -0,0 +1,791 @@
+
+<p>Click <span class="bold">Create</span> to save the new group.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<img alt="../images/facilityManagerGroupInsight.jpg" class="align-center" src="images/facilityManagerGroupInsight.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<li>
+
+<p class="first">Right-click on the <span class="bold">Group</span> tab.</p>
+
+<p>Select the <span class="bold">New User</span> entry.</p>
+
+<p>Fill in the details.</p>
+
+<p>Click to select if group owner or administrator.</p>
+
+<p>Add groups to make the user a member of them.</p>
+
+<p>Click <span class="bold">Create</span> to save the new user.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/facilityManagerUserInsight.jpg" class="align-center" src="images/facilityManagerUserInsight.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+</ol>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h2><a class="anchor" id="remove"></a>Removing a user from a group</h2>
+
+<p>This requires using the OMERO.web Admin tool.</p>
+
+<p>There are two ways to do this.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h3>OMERO.web</h3>
+
+<ol>
+
+<li>
+
+<p class="first">Click on <span class="bold">Admin</span> in the top toolbar.</p>
+
+<p>Click on the <span class="bold">Groups</span> tab.</p>
+
+<p>Click on the <span class="bold">Edit Group</span> button.</p>
+
+<p>Click on the <span class="bold">x</span> to the right of the user's name in the members or owners list.</p>
+
+<p>Click <span class="bold">Save</span> to complete the change.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/facilityManagerRemoveUser.jpg" class="align-center" src="images/facilityManagerRemoveUser.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+or
+
+<li>
+
+<p class="first">Click on the <span class="bold">Users</span> tab.</p>
+
+<p>Click on the <span class="bold">Add new User</span> button.</p>
```

Comment:
How is this 'removing a user from a group'? 
You're got the screenshots the wrong way round - edit user first in the groups tab section, and then the edit group screenshot in what should be the edit user tab description but has the wrong text